test/versioned/elastic/elasticsearch.tap.js

```diff
@@ -331,9 +332,13 @@ test('Elasticsearch instrumentation', { timeout: 20000 }, (t) => {
       const transaction = agent.getTransaction()
```

Comment:
I didn't catch these but the first arg to the callback to runInTransaction is the transaction so you can update all these tests to just use that instead of trying to get it